### DISCUSSION TITLE
Coop related requests

### DISCUSSION THREAD

**Discussion by kristophergeo** [BG-Software-LLC/SuperiorSkyblock2#1687] [Pending Feature Requests]
I am creating my skyblock server like non other skyblock server. It's basically an island per player and the island owner will be able to add coop members rather than members in their team. Ik that they could technically use the /island coop but i wanna disable that command since i wanna make the coop feature unlockable on a specific level and make the leaders add coop members from a menu in deluxemenus. To make my vision come true tho i need these additional things to be added, so if anyone knows how to code them i will really appreciate it.

**Commands:**
1) /island admin addcoop <player-name / island-name> <target-name>
2) /island admin removecoop <player-name / island-name> <target-name>

**Placeholders:** 
1) superior_island_coop_<#> - It will get a name of an island coop in an island. (similar to how the superior_island_member_<#> is)
2) superior_island_coop_online - It will get the amount of online members of an island  (similar to how the superior_island_team_size_online works)

**Comment by Oribuin** [comment]
I can't compile it but, I [think] i added this here (https://github.com/Oribuin/SuperiorSkyblock2/commit/3e110361639bfa47fd6bd3b73898bce2eed36bf6)
